### ISSUE TITLE
[RFE] Store Copy verbosity

### ISSUE THREAD

**Issue by bcdurden** [hauler-dev/hauler#145]
Currently, when using the store subcommand to copy the store to an external registry, there is no stdout to detail any kind of progress.

Default behavior should show something along the lines of the activity either being performed or having been completed such as 
```console
Pushed Image:xzy to registry://a.b.c
Pushed Chart:xyz to registry://a.b.c
Verified signature of Image:xyz in registry://a.b.c
```

**Comment by amartin120** [comment]
`hauler store copy` is `cosign load` but in our bulk version that just points cosign to the local store directory and it does all of the work versus calling `cosign load` for each individual image.  I should be able to route cosign's logging to hauler though.  Let me check into that.

**Comment by amartin120** [comment]
completed here.  https://github.com/rancherfederal/hauler/pull/146

to be included with 0.4.1 release.
